### PR TITLE
Works section added

### PR DESCRIPTION
**Pull request by Ogaga01**
Modified index.html to include work
Modified style.css
Added necessary images

### CODE REVIEW

**Review by Mre55 — CHANGES_REQUESTED** [review]
Hi @Ogaga01,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request :negative_squared_cross_mark:

You have submitted a project, that doesn't have a proper README.md file. You can use [this template](https://github.com/microverseinc/readme-template/blob/master/README.md) and edit.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -25,32 +25,91 @@ <h2>Software Developer</h2>
...
+                    <li>Terminal</li>
+                </ul>
             </section>
             <section class="social">
                 <img class="divider" src="./images/Vector4.svg" alt="divider">
                 <img class="logos" src="./images/SocialMedia.svg" alt="SocialMedia">
             </section>
+            <section class="works">
+                <img class="mywork" src="./images/Element-Right.svg" alt="my works">
+                <h1 class="recent">My Recent Work</h1>
```
Comment (mahbubul14) [inline index.html:62]: - [x] Your "My Recent Work" doesn't match with the Figma design. Please fix it.

Your design             |  Figma Design
:-------------------------:|:-------------------------:
y1 | f1

File: README.md [hunk 2]
```diff
@@ -1,2 +1,61 @@
...
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+
+### Install
+https://github.com/Ogaga01/mobile-version-portfolio-project.git
+cd mobile-version-portfolio-project
+
```
Comment (mahbubul14) [inline README.md]: - [x] Please update the "Getting Started" section content like how to install and run the project locally. Please work on that. :writing_hand:

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @Ogaga01 ,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] Check the comments under the review for more suggestions. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linters error.
:heavy_check_mark: Github flow used properly.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

- [ ] Please use [CSS Media Queries](https://www.w3schools.com/css/css3_mediaqueries.asp) for your mobile version. It will make your website more responsive and user friendly.

- [ ] For readability, please add two spaces of indentation. Do not use the tab key. You did it throughout your code in the `HTML` file. Please fix it. [For the HTML & CSS best practices ](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)we should follow the rules.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: README.md (hunk 2 above)
Comment (Ogaga01) [reply README.md]: @mahbubul14 Can you please explain this further. I don't understand this

**Review by zainsadaqat — APPROVED** [review]
# Hey @Ogaga01 😃 🥳 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### [Optional]

Your project images aren't showing up. Kindly fix them

Screen Shot 2022-03-19 at 1 28 37 PM


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**